### ISSUE TITLE
Accessibility:MAS40B:Name and Help text values are empty for the top level menu

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/vscode#17172]
VSCode Version:1.6.1
OS Version:1607

**Steps to Reproduce:**
1.Open visual studio code editor.
2.Hit 'Alt+F' to open "File" menu bar option.

**Expected Result:**
Name and help text value should be present in the  inspect tool for the "File" menu bar.

**Actual Result:**
Name and help text values are empty for the "File" menu bar option.
Note1: The same issue is observed in other (Edit,View,Go,Help) menu bar options.
Note2: Shortcut keys for "Command Pallete" are "f1" or "Ctrl+Shift+P".
Note3: Inspect and Narrator Buddy Information is attached 

**User Impact:**
Screen reader will not recite the name of the menu bar name for the blind user accessing it.

![inspectinformation_visualstudiocodeeditor_filemenubar](https://cloud.githubusercontent.com/assets/22956130/21183763/7289a2b4-c22f-11e6-8117-3f64e215fc78.jpg)
![nvda announces as menu submenu for file menu bar](https://cloud.githubusercontent.com/assets/22956130/21183769/76e08e0e-c22f-11e6-96ea-fd51323c7728.jpg)

**Comment by WiproPSAccTest** [comment]
Title has been changed to a common format to perform data population.

**Comment by bpasero** [comment]
We should support this with the new custom menu implementation.

**Comment by sbatten** [comment]
The user impact section of this is resolved, but I still don't see help text using the inspect tool. need more info to determine if this is resolved like #51841

**Comment by sbatten** [comment]
Moving to August as we await clarification from @WiproPSAccTest

**Comment by AccessibilityTestingTeam-TCS** [comment]
As per mentioned in link (https://code.visualstudio.com/updates/v1_25#_custom-title-bar-and-menus-for-windowslinux), the enhancements ex- menu bar accessible using arrow keys etc. work.

This issue still reproduces with the updated setting window.titleBarStyle : custom. Screen reader still does not read the name of the menu after pressing Alt + F  (For opening menu as shown below) 

![menuissueusingkeros](https://user-images.githubusercontent.com/39204909/44202788-8a146d80-a16a-11e8-8cbd-d56181e3c8b4.png)

Verified in below version of VS Code User Setup:

![aboutvscode](https://user-images.githubusercontent.com/39204909/44202803-98628980-a16a-11e8-81b0-17c4ec6e35f0.png)

**Comment by sbatten** [comment]
@AccessibilityTestingTeam-TCS 

In my branch, I've updated the structure to look like the following,
![image](https://user-images.githubusercontent.com/6561887/44555387-c0f9fe80-a6e9-11e8-88be-5c709a71ca91.png)


Though the name is present, it still does not announce the title of the menu when the user hits 'alt+f' as the focused item is the first menu item. However, if the user hits alt to focus the menu (indicating they want to learn the menu structure as opposed to already knowing where they are going by hitting alt+f) the screen reader reads the menu name and the shortcut. Does this meet the requirements?

**Comment by AccessibilityTestingTeam-TCS** [comment]
While we press Alt key, AT reads name and shortcuts for all menu items and navigates them using arrow keys, seems this fulfills the needed requirements.

**Comment by sbatten** [comment]
Thanks for getting back. Closing as resolved.
